src/test/mochitest/browser_dbg-breakpoints-cond.js

```diff
@@ -81,4 +81,12 @@ add_task(async function() {
   bp = findBreakpoint(dbg, "simple2", 5);
   is(bp.condition, "1", "breakpoint is created with the condition");
   assertEditorBreakpoint(dbg, 5, true);
+
+  //right click breakpoint in breakpoints list
+  rightClickElement(dbg, "breakpointItem", 3)
+  // select "remove condition";
+  selectMenuItem(dbg, 9);
+  await waitForDispatch(dbg, "SET_BREAKPOINT_CONDITION");
```

Comment:
this is probably safe, but i'm paranoid with timing issues

```suggestion
  const bpCondition = waitForDispatch(dbg, "SET_BREAKPOINT_CONDITION");
  selectMenuItem(dbg, 9);
  await bpCondition;
```